requirements.txt

```diff
@@ -1 +1 @@
-numpy>=1.5
```

Comment:
Right, good point. I think given that `setup.py` would have borked on version 1.5 it's probably fine (I wonder if we do actually run on 1.5...)